### DISCUSSION TITLE
FILWallet

### DISCUSSION THREAD

**Discussion by ohash-ai** [filecoin-project/community#232] [Developers]
Digital asset wallet focusing on FIL
- https://filwallet.io 
Welcome everyone to download and experience.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
@jennijuju is this officially a partner of PL as claimed on the website?

i am missing the code for this wallet in the github repo provided on the page @pengxiankaikai

**Reply by jennijuju** [reply]
I don’t think we have a partnership with the wallet.
